src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ReferralServiceTest.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.service
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.DraftReferral
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import java.time.LocalDate
+
+class ReferralServiceTest {
```

Comment:
please can we add tests which specify that certain fields (namely 'id' and 'created') cannot be overwritten by updates?

Reply:
done in 82211c3015da4b02b8c2a6a035563e856c102969
```
@Test
fun `update cannot overwrite identifier fields`() { ... }
```